src/sentry/models/projectownership.py

```diff
@@ -25,6 +25,7 @@ class ProjectOwnership(Model):
     date_created = models.DateTimeField(default=timezone.now)
     last_updated = models.DateTimeField(default=timezone.now)
     is_active = models.BooleanField(default=True)
+    codeowners_auto_sync = models.BooleanField(default=True, null=True)
```

Comment:
@getsentry/owners-migrations is it ok to keep the `default=True` here?

Reply:
Yep, this is good since we don't add the default in postgres.